apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -681,6 +693,18 @@ private static List<Long> getSnapshotIds(Operations ops, String tableName) {
         .collect(Collectors.toList());
   }
 
+  private static List<String> getSnapshotOperations(Operations ops, String tableName) {
```

Comment:
```suggestion
  private static List<String> getSnapshotOperationTypes(Operations ops, String tableName) {
```

Reply:
I preferred getSnapshotOperations to be consistent with getSnapshotIds and also the field name is **operations** and without the type suffix. I'm okay to change if you prefer **getSnapshotOperationTypes**. LMK pls.